test/helpers/utils.test.js

```diff
@@ -1,10 +1,42 @@
-const { expect } = require('chai');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const nock = require('nock');
 const { getRequestEvent } = require('../support');
-const { getHost } = require('../../lambdas/helpers/utils');
+const { getHost, httpsGet } = require('../../lambdas/helpers/utils');
 
-describe('getHost', () => {
-  it('fetches host from request', () => {
-    const { request } = getRequestEvent().Records[0].cf;
-    expect(getHost(request)).to.equal('o-owner-r-repo.sites-test.federalist.18f.gov');
+chai.use(chaiAsPromised);
+const { expect } = chai;
+
+describe('utils', () => {
+  afterEach(() => {
+    nock.cleanAll();
+  });
+  context('getHost', () => {
+    it('fetches host from request', () => {
+      const { request } = getRequestEvent().Records[0].cf;
+      expect(getHost(request)).to.equal('o-owner-r-repo.sites-test.federalist.18f.gov');
+    });
+  });
+
+  context('httpsGet', () => {
+    it('promisify https.get - resolve', async () => {
+      const endpoint = 'https://example.gov/index.html';
+      const url = new URL(endpoint);
+      nock(`${url.protocol}//${url.hostname}`)
+        .get(url.pathname)
+        .reply(200, 'Hello World', { 'test-header': 'testHeader' });
+      expect(httpsGet(url)).to.eventually.deep.equal({
```

Comment:
we don't need `chai-as-promised` anymore, just `await` to get the response and assert against it like you do elsewhere.